README.md

```diff
@@ -15,3 +15,39 @@ This repository houses the open-source components of the Mattermost Integrations
 ## Get Involved
 
 - [Join the discussion on Integrations Marketplace](https://community.mattermost.com/core/channels/marketplace)
+
+## Developing
+
+### Environment Setup
+
+1. Install [Go](https://golang.org/doc/install)
+
+### Building
+
+Simply run the following:
+
+```
+# Make sure Go modules is turned on
+$ export GO111MODULE=on
+$ go install ./cmd/marketplace
```

Comment:
As an aside, I rarely `go install` while /developing/, preferring instead to just `go run ./cmd/marketplace` and get the benefit of one less command while iterating. Would it make sense to distinguish developer from production environments?

Reply:
Make sense, any benefit in keeping them separate? thinking of just combining the two, since there is no variation in how server is invoked in development vs production environments.